src/test/java/by/issoft/tests/AutoCreatedWishlistTest.java

```diff
@@ -0,0 +1,63 @@
+package by.issoft.tests;
+
+import by.issoft.pagefactory.LoginPage;
+import by.issoft.pagefactory.MyWishlistsPage;
+import by.issoft.pagefactory.ProductPage;
+import by.issoft.pagefactory.SearchPage;
+import by.issoft.utils.MyTestWatcher;
+import by.issoft.utils.TestBase;
+import io.qameta.allure.Description;
+import io.qameta.allure.Step;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openqa.selenium.By;
+
+
+@TestMethodOrder(OrderAnnotation.class)
+@ExtendWith(MyTestWatcher.class)
+public class AutoCreatedWishlistTest extends TestBase {
+
+
+    @Test
+    @Step("User can add product to auto-created Wishlist")
+    @Description("Auto wishlist is successfully created, when User adds product for the first time")
+    @Order(1)
+    void userShouldBeAbleToAddProductToAutoCreatedWishlist() {
+        LoginPage loginPage = new LoginPage(driver);
+        driver.get("http://automationpractice.com/index.php?controller=authentication&back=my-account");
```

Comment:
It is not good to switch between pages using direct URls. The right option is to go to the page by clicking on the links/buttons on the page